tests/kafkatest/tests/streams/streams_upgrade_test.py

```diff
@@ -50,14 +52,21 @@ def __init__(self, test_context):
         self.leader = None
```

Comment:
Unrelated to this PR. Nit: update comment in L41/43 above: `Metadata version was bumped in 0.10.1.0` 

It was also dumped in 2.0 release (that also added version probing)

Reply:
I don't follow, I should add `metadata version dumped in 2.0 release` then?

Reply:
Something like this. Just for completeness (or maybe remove the comment, because we have corresponding variable anyway?)

Reply:
ack

Reply:
`dropped` -> `bumped` ?

Reply:
ack